File: GeradorClasses-POO2/output/Pessoa.py
class Pessoa:

	def __init__ (self, nome=None, idade=None, email=None):
		self.__nome=nome
		self.idade=idade
		self._email=email

	def get_nome(self):
		 return self.__nome

	def get_idade(self):
		 return self.idade

	def get_email(self):
		 return self._email

	def set_nome(self, nome):
		self.__nome = nome

	def set_idade(self, idade):
		self.idade = idade

	def set_email(self, email):
		self._email = email

	def __demitir(self):
		return None

	def explodir(self):
		return None

	def _matar(self):
		return None

File: GeradorClasses-POO2/output/Pessoa.java
public class Pessoa {

	private String nome;
	public int idade;
	protected String email;

	public String getNome(){
		 return this.nome;
	}

	public int getIdade(){
		 return this.idade;
	}

	public String getEmail(){
		 return this.email;
	}

	public void setNome(String nome){
		 this.nome= nome;
	}

	public void setIdade(int idade){
		 this.idade= idade;
	}

	public void setEmail(String email){
		 this.email= email;
	}

	public Pessoa(String nome, int idade, String email){
		this.nome= nome;
		this.idade= idade;
		this.email= email;
	}

	private boolean demitir(){
		return null;
	}

	public void explodir(){
		return null;
	}

	protected void matar(){
		return null;
	}

}